includes/admin/downloads/metabox.php

```diff
@@ -806,20 +810,27 @@ function edd_render_file_row( $key, $args, $post_id, $index ) {
 		'thumbnail_size' => null,
 	) );
 
-	$prices           = edd_get_variable_prices( $post_id );
-	$variable_pricing = edd_has_variable_prices( $post_id );
-	$variable_display = $variable_pricing ? '' : ' style="display:none;"';
-	$variable_class   = $variable_pricing ? ' has-variable-pricing' : ''; ?>
+	$prices           		= edd_get_variable_prices( $post_id );
+	$variable_pricing 		= edd_has_variable_prices( $post_id );
+	$variable_display 		= $variable_pricing ? '' : ' style="display:none;"';
+	$variable_class   		= $variable_pricing ? ' has-variable-pricing' : '';
+	$custom_file_options 	= isset( $wp_filter['edd_download_file_option_row'] ) ? true : false; ?>
```

Comment:
```suggestion
	$custom_file_options = has_action( 'edd_download_file_option_row' );
	?>
```
And the global can be removed from line 803. Also please fix the alignment/spaces in the previous variables (some tabs have gotten mixed in).